lightly/api/api_workflow_collaboration.py

```diff
@@ -0,0 +1,76 @@
+from os import access
+from typing import List
+
+from lightly.openapi_generated.swagger_client.models.create_entity_response import CreateEntityResponse
+from lightly.openapi_generated.swagger_client.models.dataset_create_request import DatasetCreateRequest
+
+from lightly.openapi_generated.swagger_client.models.shared_access_config_create_request import SharedAccessConfigCreateRequest
+from lightly.openapi_generated.swagger_client.models.shared_access_config_data import SharedAccessConfigData
+from lightly.openapi_generated.swagger_client.models.shared_access_type import SharedAccessType
+
+from lightly.openapi_generated.swagger_client.models.dataset_data import DatasetData
+from lightly.openapi_generated.swagger_client.rest import ApiException
+from lightly.openapi_generated.swagger_client.models.dataset_type import DatasetType
+
+class _CollaborationMixin:
+
+    def share_dataset(self, dataset_id: str, user_emails: List[str]):
```

Comment:
the naming of the function feels like one could "add to" the list. Maybe rename this one to `share_dataset_only_with` and add another fn which one could add more users (first get the old stuff and then concat) like `share_dataset` or `share_dataset_with`.

Reply:
I would not add two different methods and just make it clear to the user what this method does. If we add a method to add individual users we should also add one to remove individual users.
I thought `share_dataset` is like sharing something once (so you would add everyone that should get access). But I see your point. I'll go with `share_dataset_only_with`

Reply:
okeydokey